### PR TITLE
Add the ability to add custom log handlers to the root logger

### LINKED ISSUE #17480: quarkus.log.console.enable disable file log handlers
## Describe the bug
with the following application.properties
```
quarkus.package.type=uber-jar
quarkus.log.level=INFO
quarkus.log.console.enable=false
quarkus.log.console.level=INFO

quarkus.log.handler.file."CONSOLE_MIRROR".enable=true
quarkus.log.handler.file."CONSOLE_MIRROR".path=quarkus.log

quarkus.log.category."".handlers=CONSOLE_MIRROR
```
when running the uber jar (it's working file with quarkus:dev) 
 the file quarkus.log is created but is empty.

This was working prior quarkus 1.13

## To Reproduce

https://github.com/rmanibus/quarkus_17480

1.  mvn install
2. java -jar target/code-with-quarkus-1.0.0-SNAPSHOT-runner.jar
 -> application does not log to quarkus.log
3. java -Dquarkus.log.console.enable=true -jar target/code-with-quarkus-1.0.0-SNAPSHOT-runner.jar
 -> application log to quarkus.log

### PR DESCRIPTION
**Pull request by geoand**
Resolves: #17480

### CODE REVIEW

**Comment by geoand** [comment]
No category is needed, it would be something like:

```properties
quarkus.log.handler.file.CONSOLE_MIRROR.enable=true
quarkus.log.handler.file.CONSOLE_MIRROR.path=quarkus.log

quarkus.log.handlers=CONSOLE_MIRROR
```

And yes, if we do add this, the docs will need an update

**Comment by geoand** [comment]
If it is still an problem, best open a new issue

**Comment by geoand** [comment]
@dmlloyd is this what you had in mind for https://github.com/quarkusio/quarkus/issues/17480?

File: core/runtime/src/main/java/io/quarkus/runtime/logging/LogConfig.java [hunk 1]
```diff
@@ -101,4 +103,12 @@ public final class LogConfig {
     @ConfigItem(name = "filter")
     @ConfigDocSection
     public Map<String, CleanupFilterConfig> filters;
+
+    /**
+     * The names of the handlers to link to this category.
+     * These handlers are defined in consoleHandlers, fileHandlers or syslogHandlers
+     * and they added additionally to whatever handlers result from the root configuration.
+     */
```
Comment (dmlloyd) [inline core/runtime/src/main/java/io/quarkus/runtime/logging/LogConfig.java:111]:
```suggestion
    /**
     * The names of additional handlers to link to the root category.
     * These handlers are defined in consoleHandlers, fileHandlers or syslogHandlers.
     */
```

**Review by dmlloyd — APPROVED** [review]
Yep, looks good to me, with one minor comment about the docs.

**Comment by geoand** [comment]
Thanks.

PR updated and taken out of draft

**Pull request merged**